routing/route.hpp

```diff
@@ -64,15 +64,30 @@ class RouteSegment final
     uint8_t m_maxSpeedKmPH = 0;
   };
 
+  struct RoadNameInfo
+  {
+    bool m_isLink;
```

Comment:
- put to the end (after strings)
- bool m_isLink = false; and you don't need constructor

Reply:
OK